### ISSUE TITLE
App Redesign: new hero section (no vaults)

### ISSUE THREAD

**Issue by jahabeebs** [open-dollar/od-app#291]
## New hero section *when no vaults*

- Background should be the new blue gradient, include new vault graphic and button should be deep blue  

Current design as of 3/22 but reference latest

#### Screen 1: Hero & connect wallet

![image](https://github.com/open-dollar/od-app/assets/69639595/1dc87578-2150-4434-9bac-6f977a37c05a)

#### Screen 2: Connecting Wallet

![image](https://github.com/open-dollar/od-app/assets/69639595/74f370af-bbb2-4899-9d4f-cb8670af9930)

#### Screen 3: Create Vault facilitator

![image](https://github.com/open-dollar/od-app/assets/69639595/6b30ead2-39ff-431e-ab9e-9af6b2909e2b)

#### Screen 4: Create Vault

![image](https://github.com/open-dollar/od-app/assets/69639595/7a899658-7c78-4249-8f66-b5134d35565e)

The main moving piece here is the stepper, connecting the wallet, and creating a facilitator is simple signatures in the wallet.

latest designs as of 4/02:

https://www.figma.com/file/EkcMRZhPra8NezYKt4zn7s/OD-testnet-app?type=design&node-id=678-1010&mode=design&t=QjIydVLUL2op02il-0

**Comment by alekseevaiana** [comment]
Hi @Seroxdesign, I reviewed the code base today and have outlined a plan and some questions below. Let's discuss these in our morning meeting. 
1. Should we create components specifically for mobile screens? If so, how should the stepper component be adapted for mobile use?
2. There are 3 steps and screens, but I need to figure out how to make additional step which has a different layout. 
3. Consider changing the layout for each step to accommodate two different designs.
4. I need to change button styling, including main colors and borders. However, access to the design page is required to view all button variations.
5. Style cookie wrapper 
6. Make Stepper component: do we want to use library for this kind of components? How do we choose a library so it will be safe? I couldn't find a library in a package.json that already in use. Also I took a look at the most popular libraries, and steppers look different everywhere. But I can make my own stepper, don’t think that it will take long.
7. Changing background. I see there is gradient and a picture. Hope that I can use it from Figma, but need to have access first.
